ui/library/package.json

```diff
@@ -46,7 +46,7 @@
     "mini-css-extract-plugin": "2.4.3",
     "mocha": "8.4.0",
     "prettier": "2.3.2",
-    "react-pdf": "5.3.2",
+    "react-pdf": "5.7.2",
```

Comment:
Do you also need to update ` @types/react-pdf`? if there is a new version available

Reply:
It seems working without updating `@types/react-pdf` but it's better to update it as well. Will also update the package to the newest version `v5.7.1`. https://www.npmjs.com/package/@types/react-pdf

Reply:
I end up not updating `@types/react-pdf` since it gives me weird type errors.